DeGruyter.js

```diff
@@ -182,35 +195,29 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "journalArticle",
+				"title": "The Midterm Landslide of 2010: A Triple Wave Election",
 				"creators": [
 					{
-						"firstName": "James E.",
 						"lastName": "Campbell",
-						"creatorType": "author"
-					}
-				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [
-					{
-						"title": "Full Text PDF",
-						"mimeType": "application/pdf"
-					},
-					{
-						"title": "Snapshot"
+						"creatorType": "author",
+						"firstName": "James E."
 					}
 				],
-				"title": "The Midterm Landslide of 2010: A Triple Wave Election",
-				"publicationTitle": "The Forum",
-				"volume": "8",
+				"date": "2011",
+				"DOI": "10.2202/1540-8884.1405",
+				"ISSN": "1540-8884",
+				"abstractNote": "Democrats were trounced in the 2010 midterm elections. They lost six seats in the U.S. Senate, six governorships, and about 700 seats in state legislatures. Compared to 2008, Democrats lost 64 seats in the House and Republicans regained their House majority. The Republican majority elected in 2010 was the largest number of Republicans elected since 1946. The analysis finds that Republican seat gains resulted from the receding of the pro-Democratic waves of 2006 and 2008 as well as the incoming pro-Republican wave of 2010. Voters rejected Democrats in 2010 for their failure to revive the economy, but also for their advancement of the national healthcare reform and other liberal policies. The analysis speculates that Democrats are likely to gain House seats and lose Senate seats in 2012. Finally, President Obama’s prospects of re-election have probably been improved because of the Republican gains in the 2010 midterm.",
 				"issue": "4",
+				"journalAbbreviation": "for",
```

Comment:
Well, we see these journal codes in the urls from de Gruyter, e.g. 
```
http://www.degruyter.com/view/j/for.2011.8.4_20120105083457/for.2011.8.4/for.2011.8.4.1405/for.2011.8.4.1405.xml?format=INT
```
However, I guess that is not what we want to save in `journalAbbreviation`. I will remove them.